.gitignore

```diff
@@ -21,6 +21,5 @@ coverage
 .grunt
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
-build/
-dist/
+public/bundle.js
```

Comment:
You may have to also untrack the file in VS code/terminal, if simply adding it to gitignore doesn't stop it from being tracked, check out this link: http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/